Steering.md

```diff
@@ -0,0 +1,29 @@
+# iTwin.js Steering Committee Charter 
```

Comment:
Do you intend to link to this document from somewhere? Currently, no one will see it unless they happen to browse the source.

Reply:
Yes, a blog post is going to be created and then promoted on various social media outlets including the Momentum event.

Reply:
an announcement blog post is being crafted and will link to this charter document.